src/chronometer.js

```diff
@@ -1,30 +1,40 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    setInterval((intervalId) => {
```

Comment:
Aquí tendrías que haber puesto `this.intervalId = setInterval...` para que se quede registrado el ID del "reloj" inicializado para que más abajo en el método `stop()` puedas para ese "reloj" por el ID que guardaríamos aquí y luego también, setInterval recibe una función **sin parámetro**